sigma/plugins/fun/xkcd.py

```diff
@@ -28,4 +40,4 @@
 
 def is_int(s):
```

Comment:
Python has built-in ways to check if something is an integer: `isinstance(val, int)`

Reply:
Fixed with 0a0b851